### PR TITLE
Add min- and max-scale flags for knative deployer

### PR DESCRIPTION
**Pull request by joshrider**
Adds `--min-scale` and `--max-scale` flags for knative deployer create options.

### CODE REVIEW

**Comment by joshrider** [comment]
Updating docs...

File: docs/riff_knative_deployer_create.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ riff knative deployer create my-image-deployer --image registry.example.com/my-i
       --ingress-policy policy   ingress policy for network access to the workload, one of "ClusterLocal" or "External" (default "ClusterLocal")
       --limit-cpu cores         the maximum amount of cpu allowed, in CPU cores (500m = .5 cores)
       --limit-memory bytes      the maximum amount of memory allowed, in bytes (500Mi = 500MiB = 500 * 1024 * 1024)
+      --max-scale int32         maximum number of replicas (default unbounded)
```
Comment (scothis) [inline docs/riff_knative_deployer_create.md]: I'd rather not see `int32` in the command output. We can either change the type to `int` or change the flag description to highlight the `number` from the description.

Comment (trisberg) [reply docs/riff_knative_deployer_create.md]: just changing to use \`number\` in the description should work fine IMO. Would like to see this change merged so we can switch to use Knative for the deployers for the demo

Comment (joshrider) [reply docs/riff_knative_deployer_create.md]: updated! sorry for the delay. was out yesterday.

**Pull request merged**